server/eclipse-project/src/main/java/us/freeandfair/corla/model/NENAssertion.java

```diff
@@ -7,6 +7,8 @@
 package us.freeandfair.corla.model;
 
 
+import au.org.democracydevelopers.raire.assertions.AssertionAndDifficulty;
```

Comment:
What's the reason for the addition of this import? I can't see where it is used.